src/libZilliqa/Zilliqa.cpp

```diff
@@ -109,6 +109,10 @@ Zilliqa::Zilliqa(const std::pair<PrivKey, PubKey>& key, const Peer& peer,
 
 {
   LOG_MARKER();
+
+  // Setting the guard upon process launch
+  Guard::GetInstance().Init();
```

Comment:
if (GUARD MODE) ??

Reply:
Have implemented the fix within init() function